pkg/csi/cinder/openstack/openstack_instances.go

```diff
@@ -2,6 +2,7 @@ package openstack
 
 import (
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
+	openstack_provider "k8s.io/cloud-provider-openstack/pkg/cloudprovider/providers/openstack"
```

Comment:
Package name should be in lower case, with no under_scores or mixedCaps, so `cpo` could be used as in other files.